docker/development.docker-compose.yml

```diff
@@ -6,6 +6,11 @@ services:
       context: ..
       dockerfile: docker/hardhat.Dockerfile
     restart: always
+    volumes:
+      - ../packages/hardhat/deployments:/src/packages/hardhat/deployments
+      - ../packages/hardhat/artifacts:/src/packages/hardhat/artifacts
+      - ../packages/hardhat/cache:/src/packages/hardhat/cache
+      - ../packages/hardhat/tasks:/src/packages/hardhat/tasks
```

Comment:
This is necessary so we can run tasks inside the container including the latest built artifacts.